tests/samples/songkick/tovestyrke.json

```diff
@@ -188,5 +188,18 @@
             ]
         }
     ],
-    "microformat": []
+    "microformat": [],
+     "dublincore": [
+         {
+             "namespaces": {
+
+             },
```

Comment:
:lipstick: Maybe normalize the indentation?

Reply:
done